src/pages/Address/AddressComp.tsx

```diff
@@ -350,17 +360,26 @@ export const AddressTransactions = ({
         className={styles.transactionTitleCard}
         isSingle
         rear={
-          <Radio.Group
-            className={styles.layoutButtons}
-            options={[
-              { label: i18n.t('transaction.professional'), value: 'professional' },
-              { label: i18n.t('transaction.lite'), value: 'lite' },
-            ]}
-            onChange={({ target: { value } }) => onChangeLayout(value)}
-            value={layout}
-            optionType="button"
-            buttonStyle="solid"
-          />
+          <>
+            <div className={styles.sortAndFilter}>
+              {timeOrderBy === 'asc' ? (
+                <TimeDownIcon onClick={handleTimeSort} />
+              ) : (
+                <TimeUpIcon onClick={handleTimeSort} className={styles.asc} />
```

Comment:
why `className={styles.asc}` when `timeOrderBy` is NOT `asc`

Reply:
The list is order by time, either in `asc` or `desc`

Reply:
The class name is avoided in https://github.com/Magickbase/ckb-explorer-frontend/pull/8/commits/fb5742efa90345c6b0919a490e6c4893093e9e38, which uses an attribute to indicate the status of the order. It separates data and visual style of a component by using attribute instead of class name